bom/compile/pom.xml

```
@@ -318,6 +318,25 @@
       <scope>compile</scope>
     </dependency>
 
+    <!-- dirwatcher -->
+    <dependency>
+      <groupId>org.openhab.osgiify</groupId>
+      <artifactId>io.methvin.directory-watcher</artifactId>
+      <version>0.15.1</version>
```

When resuming PR, you probably want latest version, so just a reminder - currently:
```suggestion
      <version>0.17.1</version>
```

---

Needs an upgrade in openhab-osgiify before

---

openhab/openhab-osgiify#37 was just merged. 👍